cmd/permissions-checker/permissionschecks.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os/user"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"syscall"
+)
+
+type currentPermissions struct {
+	permissions           string
+	defaultPermissions    string
+	owner                 string
+	group                 string
+	hasCorrectPermissions bool
+	hasCorrectOwner       bool
+	hasCorrectGroup       bool
+}
+
+// FindRecursive iterates in a directory showing permissions in a recursive way
+/* path [string] 											- where to look for files and directories
```

Comment:
I do not know if this is the correct way of documenting parameters. Any expert please confirm.